### PR TITLE
✏️Grammar changes to docs

### PR DESCRIPTION
**Pull request by kewbish**
Saw your post on Dev.to Listings, thought I'd stop by and help edit some docs.   

Changed:
- verb form usage
- some phrases to make docs clearer
- removed excess spaces in some places

Let me know if this is alright!

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Our Pledge
 
 In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
+contributors and maintainers pledge to make participation in our project and
```
Comment (Ousret) [inline CODE_OF_CONDUCT.md]: `CODE_OF_CONDUCT.md` is untouched from Github. Also available here [www.kernel.org](https://www.kernel.org/doc/html/latest/process/code-of-conduct.html). I prefer leaving this file untouched.

File: README.md [hunk 2]
```diff
@@ -30,54 +30,54 @@
 
 ### ❓ Why
 
-No matters your religion, IMAP4 or HTTP, you should not worries about accessing easily header and associated attributes, adjectives or values.
+No matter your religion or choice of IMAP4 or HTTP, you should not worry about easily accessing header and associated attributes, adjectives or values.
```
Comment (Ousret) [inline README.md]: I think I was misunderstood here ^^. I wanted to compare IMAP4 and HTTP as a religion. *some kind of joke*. 
What I actually wanted to say is : *No matter if you are currently building software using HTTP or IMAP communications."

File: docs/quickstart.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ headers.content_type.charset  # output: ISO-8859-1
 
 ### OneToOne, OneToMany
 
-Do not forget that headers are not 1 TO 1. One header can be repeated multiple time and attribute can have multiple value within the same header.
+Do not forget that headers are not 1 to 1. One header can be repeated multiple times and attributse can have multiple values within the same header.
```
Comment (Ousret) [inline docs/quickstart.md]: Minor typo at `attributeS`

**Review by Ousret — CHANGES_REQUESTED** [review]
Overall LGTM. I left some comments.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (kewbish) [reply CODE_OF_CONDUCT.md]: Alright - Might make a PR directly to them, then. I've changed it back.

File: README.md (hunk 2 above)
Comment (kewbish) [reply README.md]: Ah, sorry for misunderstanding. Changed to what you meant to say.

File: docs/quickstart.md (hunk 3 above)
Comment (kewbish) [reply docs/quickstart.md]: Oops, meant to say attributes.

**Comment by kewbish** [comment]
Alright, fixed the suggestions in this commit.

Let me know if there's anything else.

**Comment by Ousret** [comment]
Thank you @kewbish for your contribution.

**Pull request merged**